### ISSUE TITLE
WCAG 2.2 plans?

### ISSUE THREAD

**Issue by msgannon** [daisy/ace#413]
Does anyone know if there are plans to include WCAG 2.2 criteria to the Ace testing?
Thanks!

**Comment by danielweck** [comment]
Related: https://github.com/dequelabs/axe-core/issues/4361

**Comment by msgannon** [comment]
Thanks for all of this!

**Comment by lasselehtinen** [comment]
I was just looking into the same issue. What about the older WCAG 2.1 which is recommended even though 2.0 is the absolute minimum? I did not find a way validate against the 2.1 ruleset. Or is 2.1 support something that will come together with the Deque Axe integration?

**Comment by danielweck** [comment]
Current Ace architecture is based on the premise that the latest revision of W3C EPUB Accessibility (1.1) is supported, there is no API control to select a target version, let alone to tweak WCAG conformance requirements. Ace simply executes Deque Axe for each XHTML documents in the EPUB publication. We do fork the Deque Axe codebase and we create our own build in order to address a few EPUB-specific points, so we have some scope to adapt the overall architecture and enable more flexibility (consumer-side) ... but this would be a non-trivial undertaking as I currently understand it.

**Comment by lasselehtinen** [comment]
OK I see your point. Your target is W3C EPUB Accessibility (1.1) which supports different WCAG versions. But the new EU accessibility directive is actually requiring WCAG 2.1 or greater. This is an issue because now we cannot validate ePubs against this standard with Ace.

**Comment by danielweck** [comment]
> WCAG 2.1 or greater

Deque Axe level of support is a little bit unclear to me, when I read this:
https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md

**Comment by lasselehtinen** [comment]
Did understood the commit correctly, now different ruleset combinations are supported and we can validate against spesific WCAG version?

If that is the case, when is the next release coming out?
